tests/testTriggers.go

```diff
@@ -5912,7 +5914,7 @@ func TriggerAsyncDRVolumeOnly(contexts *[]*scheduler.Context, recordChan *chan *
 }
 
 func TriggerStorkApplicationBackup(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
-	dash.Infof("Stork Appplication Backup triggered at: %v", time.Now())
+	log.Infof("Stork Appplication Backup triggered at: %v", time.Now())
```

Comment:
if u want to post to dashboard plz use log.InfoD

Reply:
Done.